src/utils.ts

```diff
@@ -3,20 +3,20 @@ import { LICHESS_API_URL } from './constants'
 export async function fetcher({
   endpoint,
   token,
-  post,
+  method,
   body,
 }: {
   endpoint: string
   token: string
-  post?: boolean
+  method?: string
   body?: URLSearchParams
 }) {
   return (
     await fetch(`${LICHESS_API_URL}${endpoint}`, {
       headers: {
         Authorization: `Bearer ${token}`,
       },
-      method: post ? 'POST' : 'GET',
+      method,
```

Comment:
We can set the default method to `GET`, so we only need to pass other methods if necessary